common/middleware/set-primary-navigation.js

```diff
@@ -18,15 +18,24 @@ function setPrimaryNavigation(req, res, next) {
     items.push({
       text: req.t('primary_navigation.single_requests'),
       href: '/moves/requested',
-      active: req.path.endsWith('/requested'),
+      active: req.path.startsWith('/moves') && req.path.endsWith('/requested'),
+    })
+  }
+
+  if (permissions.check('allocations:view', userPermissions)) {
+    items.push({
+      text: req.t('primary_navigation.allocations'),
+      href: '/allocations',
+      active:
+        req.path.startsWith('/allocations') && req.path.endsWith('/outgoing'),
```

Comment:
Is this active check correct?

Reply:
Yep, so the allocations URLs contain the same route structure as moves to share the code around collections.

The both use: `/:base/:date/:locationId?/:view`. This is looking at the base and the last path.

For allocations, we could potentially drop the second check as we only list one type at the moment.

Reply:
but for super clarity, it was the second part of the check I was asking about

 `&& req.path.endsWith('/outgoing')`

since surely `/allocations` itself wouldn't be matched - presumably we redirect away from that so it doesn't arise